pkgs/tools/networking/wireproxy/default.nix

```diff
@@ -22,6 +22,9 @@ buildGoModule rec {
 
   vendorHash = "sha256-LBLEb2oVi5ILNtoOtmJZ7NC7hMvLZcexYAxwmb4iUBo=";
 
+  doInstallCheck = true;
+  installCheckPhase = "$out/bin/wireproxy --version";
```

Comment:
Why not `passthru.tests.version = testers.testVersion { ... };` instead?

Reply:
Done. :)